src/main/java/seedu/duke/Exercise.java

```diff
@@ -1,4 +1,31 @@
 package seedu.duke;
 
 public class Exercise {
-}
+    protected String nameOfExercise; // The name of the Exercise
+    protected Calorie caloriesBurnt;// Amount of calories burnt from exercise
+    protected int durationOfExercise = 1; // Duration of the exercises
+    protected int caloricBurnRate = caloriesBurnt.get()/durationOfExercise; // Caloric Rate of the food, if amount isnt given we will take it as a whole unit.
+    public Exercise(String nameOfExercise,Calorie caloriesBurnt, int durationOfExercise){
+        this.nameOfExercise = nameOfExercise;
+        this.caloriesBurnt = caloriesBurnt;
+        this.durationOfExercise = durationOfExercise;
+    }
+    public Exercise(String name,Calorie caloriesBurnt){
```

Comment:
Try to be consistent with the spacing between methods? And add the whitespaces in the method names, eg
` public Exercise(String name, Calorie caloriesBurnt) {`
For this class and the other classes as well.